goal_src/engine/game/projectiles.gc

```diff
@@ -869,13 +869,11 @@
    (-> obj part)
    (the-as vector (-> obj root-override root-prim prim-core))
    )
-  0
-  (none)
+  (the-as uint 0)
```

Comment:
I'd guess that this doesn't return anything.

Reply:
yeah i think i got thrown off by method 28 in `mother-spider-proj`, it seems to return a vector.  So this is either ignorable, or it needs a replace.  Everything still compiles with the methods back to `none`s though